src/main/resources/assets/languages/de_DE.json

```diff
@@ -167,6 +171,10 @@
       "info": "%1$sDu kannst deine Serversprache (sofern anwendbar) mithilfe von `~>opts language set <language code>` ändern. Deine eigene Sprache kannst du mithilfe von `~>profile lang <language code>` ändern.\n**Verfügbare Sprache:** `%2$s`\nSollte deine Sprache hier nicht mitaufgeführt sein und du hast Lust zu helfen, kannst du gerne dem Support-Server beitreten und bei der Übersetzung helfen!"
     },
     "music_general": {
+      "music_error": {
+        "timeout": "%1$sNicht in der Lage zu %2$s zu verbinden: connection timed out. Versuche die Serverregion zu ändern und versuch den Befehl erneut.",
```

Comment:
translate connection timed out to Verbindugsfehler

Reply:
*Verbindungsfehler (im good at german trust me :mmlol:)

Reply:
why does german always sound like you're screaming?
english: "Connection timed out"
german: "VERBINDUNGSFEHLER"

Reply:
Why are you meming on an actual review thats a better question

Reply:
Wouldn't connection timed out -> Zeitüberschreitung be better?

Reply:
Would also work but wouldnt be as understandable by people who don't know much about package flow